extensions/google-meet/src/index.ts

```diff
@@ -1,5 +1,11 @@
-import { open } from "@raycast/api";
+import { getMeetTab, openMeetTab } from "./helpers";
+
+import { showHUD, Clipboard } from "@raycast/api";
 
 export default async function main() {
-  await open("https://meet.google.com/new");
+  await openMeetTab();
+  const meetTab = await getMeetTab();
+
+  await Clipboard.copy(meetTab);
+  await showHUD(`Your meet link was copied to clipboard: ${meetTab}`);
```

Comment:
Usually, we keep the HUD message short and follow the same pattern: `Copied XYZ to the clipboard`. I would suggest to follow the same pattern for consistency.

```suggestion
  await showHUD("Copied meet link to clipboard");
```

Reply:
you are right, thanks for the suggestion!